build.gradle

```diff
@@ -19,29 +19,27 @@ buildscript {
         jcenter()
         google()
         gradlePluginPortal()
-//        maven { url 'https://dl.bintray.com/kotlin/kotlin-eap' }
     }
 
     dependencies {
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$KOTLIN_VERSION"
+        classpath "org.jlleitschuh.gradle:ktlint-gradle:9.2.1"
     }
 }
 
 apply plugin: 'org.jetbrains.kotlin.multiplatform'
 
 repositories {
     mavenCentral()
-//    maven { url 'https://dl.bintray.com/kotlin/kotlin-eap' }
 }
 
 group = GROUP
 version = VERSION_NAME
 
 kotlin {
     targets {
-//            macosX64("nativeCommon")
         jvm()
-    js()
+        js()
```

Comment:
We should start doing
``` 
js {
    browser()
    nodejs()
}
```
JS has this soft concept of targets sort of like the different native targets, except they don't get separate source-sets. I think by the time of the final 1.4 release it will be an error not to declare them.